provision/.env.js

```diff
@@ -1,24 +1,21 @@
-var childProcess = require('child_process');
 var chromedriver = require('chromedriver');
-var psTree = require('ps-tree');
 
 module.exports = {
-    "user"     : "admin",
-    "password" : "password",
-    before : function(done) {
-        this.chromedriverInstance = childProcess.exec('xvfb-run ' + chromedriver.path);
-        done();
-    },
-    after : function(done) {
-        if (this.chromedriverInstance != null){
-            psTree(this.chromedriverInstance.pid, function (err, children) {
-                childProcess.spawn('kill', ['-9'].concat(children.map(function (p) { return p.PID })));
-            });
-        }
-        done();
-    },
-    beforeEach: function( browser, done ) {
-        var page = browser.page.WordPressHelper();
-        page.login();
-    }
-};
```

Comment:
I need to check back on this but there was a particular reason why I was doing it like this...I just can't remember now. Should have documented that...

Reply:
When I attempted to use the previous configuration, I couldn't get the tests to run at all. @moorscode found that basic configuration which did seem to do the trick, hence why I changed it.

Reply:
Ah I remember now what the issue was: When using the plain `chromedriver` without `xvfb` the headless chrome wasn't properly working on any interaction with an input (something about [missing keycode conversion without a window manager](https://bugs.chromium.org/p/chromedriver/issues/detail?id=1018)).
Did you try provisioning with that change and run the Nightwatch tests successfully? Maybe they have meanwhile fixed this in a recent Chrome version.

Reply:
The tests weren't successful as logging in resulted in an immediate redirect to the login page. I don't believe that's related to this change.

Reply:
Okay I've looked into this and it seems as if Chrome fixed this in the meantime, so yeah, this change is ok by now.